src/TopBar/index.tsx

```diff
@@ -15,16 +15,28 @@ export interface Props {
   title?: React.ReactNode;
   customLogo?: React.ReactNode;
   onLogoClick?: Function;
+  theme?: TopBarTheme;
+}
+
+export enum TopBarTheme {
+  DEFAULT = "default",
+  PLAIN = "plain",
 }
 
 /**
  * Global page-level header component.
  */
 export class TopBar extends React.PureComponent<Props> {
+  static defaultProps: Partial<Props> = {
+    theme: TopBarTheme.PLAIN,
+  };
```

Comment:
I wonder if the default should actually be `TopBarTheme.DEFAULT`. Otherwise, when existing repos like launchpad update Dewey, their top bars will unexpectedly change

Reply:
bah - thanks for catching that! just had that in for testing